guides/advanced_guides/settings.md

```diff
@@ -0,0 +1,280 @@
+
+## Synonyms
+
+A set of words defined for an index. Synonyms are **different words that have the same meaning**, and thus are treated similarly. If either of the associated words is searched, the same results shall be displayed.
+
+`synonyms=<Object>`
+
+- `<Object>` (Object, defaults to `{}`) : `{ <String>: [<String>, <String>, ...], ... }`
+
+  An object that contains words with a list of their associated synonyms.
+
+[Learn more about synonyms](/guides/advanced_guides/synonyms.md)
+
+#### Example
+
+Suppose you have an e-commerce dataset. For an index that contains information about tops, you decide to create synonyms for `sweater` and `jumper` since these two items are very similar.
+
+```bash
+$ curl \
+  -X POST 'http://localhost:7700/indexes/tops/settings' \
+  --data '{
+      "synonyms": {
+          "sweater": ["jumper"],
+          "jumper": ["sweater"]
+      }
+  }'
+```
+
+By doing so, if you search either `black sweater` or `black jumper` for example, the **same results** will be displayed.
+
+## Stop words
+
+A set of words defined for an index. Because some words neither add semantic value nor context, you may want to ignore them from your search. Stop words are **ignored during search**.
+
+`stopWords=[<String>, <String>, ...]`
+
+- `[<String>, <String>, ...]` (Array of strings, defaults to `[]`)
+
+  An array of strings that contains the stop words.
+
+[Learn more about stop words](/guides/advanced_guides/stop_words.md)
+
+#### Example
+
+To add `the`, `a` and `an` to the stop words list, send:
+
+```bash
+$ curl \
+  -X POST 'http://localhost:7700/indexes/movies/settings' \
+  --data '{
+      "stopWords": [
+          "the",
+          "a",
+          "an"
+      ]
+  }'
+```
+
+With the settings in the example above, `the`, `a` and `an` are now ignored by the sorting algorithm if they are present in search queries.
+
+Suppose you would like to search `the mask` in a movie database. Since `the` is ignored during search, MeiliSearch will look for every movie containing `mask` and not the millions ones containing `the`. `the` is a less relevant term than `mask` and also a very frequent word in English. By adding `the` to the stop words list, MeiliSearch will ignore this word, and thus be faster to answer without losing in relevancy.
+
+## Ranking rules
+
+Built-in ranking rules that **ensure relevancy in search results**. Ranking rules are applied in a default order which can be changed in the settings. You can add or remove rules and change their order of importance.
+
+`rankingRules=[<String>, <String>, ...]`
+
+- `[<String>, <String>, ...]` (Array of strings, see default value below)
+
+  An array of strings that contains the ranking rules sorted by order of importance (arranged from the most important rule to the least important rule).
+
+Default value:
+
+```json
+["typo", "words", "proximity", "attribute", "wordsPosition", "exactness"]
+```
+
+By default, rules are executed in the following order:
```

Comment:
I am not sure we should write again the ranking rules in this guide, this also could be a source of inconsistency since we would have to change twice the text when needed

Reply:
Yes, I agree!